modules/dop/component-protocol/components/project-pipeline/myPage/tabsTable/pipelineTable/provider.go

```diff
@@ -41,6 +42,7 @@ import (
 	"github.com/erda-project/erda/modules/dop/providers/projectpipeline"
 	"github.com/erda-project/erda/modules/dop/providers/projectpipeline/deftype"
 	protocol "github.com/erda-project/erda/modules/openapi/component-protocol"
+	"github.com/erda-project/erda/pkg/limit_sync_group"
```

Comment:
pkg name prefer '-' than '_' ?

Reply:
ok, have many file use this pkg, change in other pr